dts/arm64/nxp/nxp_mimx93_a55.dtsi

```diff
@@ -322,6 +322,30 @@
 		#size-cells = <0>;
 	};
 
+	flexcan1: can@443a0000 {
+		compatible = "nxp,flexcan-fd", "nxp,flexcan";
+		reg = <0x443a0000 DT_SIZE_K(64)>;
+		interrupt-parent= <&gic>;
+		interrupts = <GIC_SPI 8 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>,
+		<GIC_SPI 9 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
```

Comment:
Please indent this line to line up with the line above, starting with `<GIC ...`.